FanGraphSpider.py

```diff
@@ -1,174 +0,0 @@
-import scrapy
-import tweepy
-import datetime
-from scrapy_splash import SplashRequest
-import csv
-from scrapy.crawler import CrawlerProcess
-
-
-tkey = 'Redacted'
-tsecret = 'Redacted'
-atoke = 'Redacted'
-asecret = 'Redacted'
-
-teamdict = {
-    'Atlanta Braves': 'Braves',
-    'Miami Marlins': 'Marlins',
-    'New York Mets': 'Mets',
-    'Philadelphia Phillies': 'Phillies',
-    'Washington Nationals': 'Nationals',
-    'Chicago Cubs': 'Cubs',
-    'Cincinnati Reds': 'Reds',
-    'Milwaukee Brewers': 'Brewers',
-    'Pittsburgh Pirates': 'Pirates',
-    'St. Louis Cardinals': 'Cardinals',
-    'Arizona Diamondbacks': 'Diamondbacks',
-    'Colorado Rockies': 'Rockies',
-    'Los Angeles Dodgers': 'Dodgers',
-    'San Diego Padres': 'Padres',
-    'San Francisco Giants': 'Giants',
-    'Baltimore Orioles': 'Orioles',
-    'Boston Red Sox': 'Red%20Sox',
-    'New York Yankees': 'Yankees',
-    'Tampa Bay Rays': 'Rays',
-    'Toronto Blue Jays': 'Blue%20Jays',
-    'Chicago White Sox': 'White%20Sox',
-    'Cleveland Indians': 'Indians',
-    'Detroit Tigers': 'Tigers',
-    'Kansas City Royals': 'Royals',
-    'Minnesota Twins': 'Twins',
-    'Houston Astros': 'Astros',
-    'Los Angeles Angels': 'Angels',
-    'Oakland Athletics': 'Athletics',
-    'Seattle Mariners': 'Mariners',
-    'Texas Rangers': 'Rangers',
-}
-
-#Builds URLs to be scraped from team name and date. Team Name Gathered from MLB SPider.
-starturls = []
-dt = datetime.date.today().strftime('%Y-%m-%d')
-with open('schedule.csv', 'r') as input:
-    reader = csv.reader(input)
-    for item in reader:
-        # Only adds item to url if time is later than scheduled game time
-        if item[0] in teamdict.keys() and item[1] < datetime.datetime.now().strftime('%H:%M'):
-            starturls.append('https://www.fangraphs.com/livewins.aspx?date=%s&team=%s&dh=0&season=%s' % (dt, teamdict[item[0]], dt[:4]))
-
-
-class FanGraphSpider(scrapy.Spider):
-    name = 'FanGraphSpider'
-    start_urls = starturls

```

Comment:
This would look a little more natural to my eyes if the starturls stuff from above was wrapped in a function, and then the line was `start_urls = build_start_urls()`. Nit pick

Reply:
I like it. Clarity! Changed it.